### PR TITLE
Features/movie info creating

### PR DESCRIPTION
**Pull request by Mateusz-95**
I added movie info but also moviePeople, thanks Maciej for help :)

### CODE REVIEW

File: src/common/Tiles/PeopleTile/styled.js [hunk 1]
```diff
@@ -3,56 +3,70 @@ import { ReactComponent as MissingPhotoIcon } from "../../../images/missingProfi
...
-    max-height: auto;
-    display: grid;
-    grid-template-rows: auto, repeat(2, 1fr);
-    padding: 16px;
-    border-radius: 5px;
-    box-shadow: 0px 4px 12px 0px ${boxShadowColor};
+  background-color: ${({ theme }) => theme.color.white};
+  width: 100%;
+  max-width: 208px;
+  height: 339px;
```
Comment (lenarmathias) [inline src/common/Tiles/PeopleTile/styled.js]: I think it would be better to set **height: auto;**

**Review by lenarmathias — COMMENTED** [review]
PeopleTiles have inconsistent alignment and size. They should all start from the same line.

File: src/common/Tiles/PeopleTile/styled.js (hunk 1 above)
Comment (Mateusz-95) [reply src/common/Tiles/PeopleTile/styled.js]: Maybe you didn't pull all the changes? It displays correctly on my end.

**Review by lenarmathias — APPROVED** [review]
Well done, now it looks good!

**Pull request merged**